cli/command/stack/kubernetes/remove.go

```diff
@@ -20,8 +20,7 @@ func RunRemove(dockerCli *KubeCli, opts options.Remove) error {
 		fmt.Fprintf(dockerCli.Out(), "Removing stack: %s\n", stack)
 		err := stacks.Delete(stack)
 		if err != nil {
-			fmt.Fprintf(dockerCli.Out(), "Failed to remove stack %s: %s\n", stack, err)
-			return err
+			return fmt.Errorf("Failed to remove stack %s: %s", stack, err)
```

Comment:
nit; probably better to use;

```go
return errors.Wrapf(err, "Failed to remove stack %s", stack)
```

Can also put the `err` inside the `if`; 

```go
if err := stacks.Delete(stack); err != nil {
	return errors.Wrapf(err, "Failed to remove stack %s", stack)
}
```

Reply:
PTAL